src/services/signalr.ts

```diff
@@ -20,24 +20,23 @@ export class SignalR {
 
     public connect(options?: IConnectionOptions): Promise<ISignalRConnection> {
 
-
         let $promise = new Promise<SignalRConnection>((resolve, reject) => {
 
-
             let configuration = this.merge(options ? options : {});
 
-            if (configuration.logging) {
-                console.log(`Connecting with...`);
-                console.log(`configuration:[url: '${configuration.url}'] ...`);
-                console.log(`configuration:[hubName: '${configuration.hubName}'] ...`);
-            }
-
             try {
                 let serialized = JSON.stringify(configuration.qs);
+
                 if (configuration.logging) {
+                    console.log(`Connecting with...`);
+                    console.log(`configuration:[url: '${configuration.url}'] ...`);
+                    console.log(`configuration:[hubName: '${configuration.hubName}'] ...`);
                     console.log(`configuration:[qs: '${serialized}'] ...`);
                 }
-            } catch (err) { }
+
+            } catch (err) {
+                console.log(err);
```

Comment:
We can silently ignore serialization errors for logging. Could you remove this line ?